src/libs/kata-types/src/config/mod.rs

```diff
@@ -164,6 +160,267 @@ impl TomlConfig {
     }
 }
 
+mod drop_in {
```

Comment:
I think it's better to put the `drop_in` mod and the unit test into a new file `drop_in.rs` under the `config` folder, and declare it  in this file.

Reply:
I agree and will do!